sound/soc/soc-generic-dmaengine-pcm.c

```diff
@@ -62,6 +62,10 @@ int snd_dmaengine_pcm_prepare_slave_config(struct snd_pcm_substream *substream,
 	struct snd_dmaengine_dai_dma_data *dma_data;
 	int ret;
 
+	if (rtd->num_cpus > 1)
+		dev_warn(rtd->dev,
+			 "%s doesn't support Multi CPU yet\n", __func__);
```

Comment:
What would actually happen in such a case? Would it still work at least with just one CPU codec? Shouldn't we return an error in this and similar cases instrad?

Reply:
> What would actually happen in such a case? Would it still work at least with just one CPU codec? Shouldn't we return an error in this and similar cases instrad?

To be honest, I don't know how to test it. I should still work with one CPU. I don't want to return an error for now is because I want to know what will happen if these functions are invoked with multi cpus.